packaging/linux/aur/PKGBUILD

```diff
@@ -0,0 +1,55 @@
+# Edit on github: https://github.com/SunshineStream/Sunshine/tree/nightly/packaging/linux/aur/PKGBUILD
+# Reference: https://wiki.archlinux.org/title/PKGBUILD
+
+pkgname=@SUNSHINE_AUR_PKG@
+pkgver=@PROJECT_VERSION@@SUNSHINE_SUB_VERSION@
+pkgrel=1
+pkgdesc="@PROJECT_DESCRIPTION@"
+arch=('x86_64' 'i686')
+url=@PROJECT_HOMEPAGE_URL@
+license=('GPL3')
+
+depends=('boost-libs' 'ffmpeg4.4' 'libpulse' 'libevdev' 'libx11' 'libxcb' 'libxfixes' 'libxrandr' 'libxtst' 'openssl' 'opus' 'udev')
+makedepends=('git' 'cmake' 'boost' 'make')
+optdepends=('cuda' 'libdrm' 'libcap')
```

Comment:
Should we add dependency version requirements? https://wiki.archlinux.org/title/PKGBUILD#depends

Reply:
Also, descriptions of when the optional dependencies are needed: https://wiki.archlinux.org/title/PKGBUILD#optdepends

Reply:
![image](https://user-images.githubusercontent.com/42013603/174691910-1321bb1d-fee0-4f55-b433-05470d4f916f.png)

@sitiom care to elaborate on this? should it be required or optional?

Reply:
Optional because it enables the discoverability of Sunshine on the local network? Or should that be required?

Reply:
It would make more sense as a required dependency.

Reply:
Added `avahi`

Reply:
> Should we add dependency version requirements? https://wiki.archlinux.org/title/PKGBUILD#depends
Possibly, however on Arch everyone is supposed to be up to date with latest packages anyway so it probably doesn't add much.

> Also, descriptions of when the optional dependencies are needed
Yes I agreed that would be useful.